pkg/apis/pipeline/v1alpha1/cluster_task_types.go

```diff
@@ -70,3 +52,15 @@ type ClusterTaskList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []ClusterTask `json:"items"`
 }
+
+func (t *ClusterTask) TaskSpec() TaskSpec {
+	return t.Spec
+}
+
+func (t *ClusterTask) TaskMetadata() metav1.ObjectMeta {
+	return t.ObjectMeta
+}
+
+func (t *ClusterTask) Copy() TaskInterface {
+	return t.DeepCopy()
+}
```

Comment:
did these get moved for a functional reason or just for code organization?

Reply:
@bobcatfish just for code organization :angel: